start-team.py

```diff
@@ -0,0 +1,85 @@
+import subprocess
+import os
+import signal
+import threading
+import logging
+import argparse
+import check_requirements
+
+
+# Set up logging
+logging.basicConfig(level=logging.DEBUG, format='%(asctime)s - %(levelname)s - %(message)s')
+
+def run_server_script(args):
+    # Start the server.py script as a new process group
+    process = subprocess.Popen(
+        ['python3', 'server.py', '--g-port', args.g_port],
```

Comment:
Since we are using python, why create a separate process?

Reply:
to make it similar to start-team.sh